TESTING.md

```diff
@@ -0,0 +1,22 @@
+# Testing environment
+
+Install the project as described [here](./README.md).
+
+# Manual testing
+
+Manual testing is done on Storybook branches available in each PR [See the Storybook](https://v1--613a8e945a5665003a05113b.chromatic.com/)
```

Comment:
My idea on how to extend this part. This image might be helpful
<img width="913" alt="image" src="https://user-images.githubusercontent.com/8076673/188857010-1ff3fb78-4762-49a2-bcff-d999ce68cb31.png">

```suggestion
## Manual testing

Manual testing is done using [Storybook](https://storybook.js.org/) and a separate instance is published for every PR (feature branch) using [Chromatic](https://www.chromatic.com/). For each component, the first story allows using [Controls](https://storybook.js.org/docs/react/essentials/controls) and [Actions](https://storybook.js.org/docs/react/essentials/actions), which simplify the process of testing most of the scenarios. The remaining stories work as a sort of documentation that can be used in order to compare implementation with designs. Other useful add-ons:
* [Measure & Outline](https://storybook.js.org/docs/react/essentials/measure-and-outline)
* [Theme Switcher](https://storybook.js.org/addons/storybook-addon-themes)
* [Viewport](https://storybook.js.org/docs/react/essentials/viewport)

Storybook instances are currently available under the following link pattern:
https://{branch-name}--613a8e945a5665003a05113b.chromatic.com/

where {branch-name} is a branch name where `/` and other special characters should be replaced with `-`.

You can also find a direct link to the latest commit instance in the PR checks section:
{image}
```